lib/src/drawer.dart

```diff
@@ -667,7 +680,8 @@ class MNavigatorObserver extends NavigatorObserver {
     "/messcalendar": 14,
     "/messcalendar/qr": 14,
     "/groups": 15,
-    "/InSeek": 16,
+    // "/InSeek": 16,
```

Comment:
If inseek was 16 why is buynsell also 16?
It might be possible that inseek is restored in the future. So change the number to 17

Reply:
```suggestion
    "/InSeek": 16,
```